### ISSUE TITLE
[REQ] Leave timeblocks emtpy without anyshows (for example at night)

### ISSUE THREAD

**Issue by korumera** [DEFENDORe/pseudotv#15]
Don't know if many people would like this, but what I would like to see is the option to have time blocks without any shows on specific channels or all channels. For example, if you create a new channel have an option to not show any TV shows or movies from 1am till 9am.

**Comment by powdermilkman** [comment]
Could be possible to add "dead air" time blocks, maybe stream a short static animation :P

**Comment by korumera** [comment]
Yeah something like that, it's mostly because of how pseudo now runs for me it's on my pc and it goes down at night :). Tried it on docker (with 0.50) but that crashed every time i changed the channel or terminated the stream. I'll check that again with the new update.

**Comment by FangsMcWolf** [comment]
Sort of an "off the air" status, where just the logo is shown, or maybe there's a way that Plex could be told that there's nothing on at that time so it won't even allow the user to try?
